rb/poll.py

```diff
@@ -157,6 +157,15 @@ def poll(self, timeout=None):
         return rv
 
 
+def is_closed(f):
+    poller = poll()
```

Comment:
On second though i need to change it anyways. The select poller cannot detect closed sockets.